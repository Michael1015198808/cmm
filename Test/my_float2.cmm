float __add__()
{
    .e-4;
    i = 1;
    1.05e;
}
