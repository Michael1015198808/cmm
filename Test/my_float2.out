Error type B at Line 3: Something wrong with the statement.
Error type A at Line 5: Illegal FLOAT number: "1.05e"
Error type A at Line 7: Illegal FLOAT number: "5."
Error type A at Line 9: Illegal FLOAT number: ".5"
